eip-3368.md

```diff
@@ -0,0 +1,61 @@
+---
+eip: 3368
+title: Increase block rewards to 3 ETH, with 2 Year Decay to 1 ETH Scheduled
+author: Michael D. Carter (BitsBeTrippin)
+discussions-to: https://ethereum-magicians.org/t/eip-3368-block-reward-increase-w-decay-for-next-two-years/5550
+status: Draft
+type: Standard Track
+category: Core
+created: 2021-03-12
+---
+
+## Simple Summary
+Changes the block reward paid to proof-of-work (POW) miners to 3 ETH from existing 2 ETH and starts a decay schedule for next two years to 1 ETH Block Reward.
+## Abstract
+Starting with `FORK_BLKNUM` block rewards will be increased to a base of 3 ETH, uncle and nephew rewards will be adjusted accordingly. Every 3 months after, measured by EPOCH count, Block Reward drops 0.25 every quarter till Ethereum Block Reward reaches 1.0 BR per Block.
+
+## Motivation
+A sudden drop in PoW mining rewards could result in a sudden precipitous decrease in mining profitability that may drive miners to auction off their hashrate to the highest bidder while they figure out what to do with their now "worthless" hardware. If enough hashrate is auctioned off in this way at the same time, an attacker will be able to rent a large amount of hashing power for a short period of time at relatively low cost and potentially attack Ethereum. By setting the block reward to X (where X is enough to offset the sudden profitability decrease) and then decreasing it over time to Y (where Y is a number below the sudden profitability decrease), we both avoid introducing long term inflation while at the same time spreading out the rate that individual miners cross into the unprofitable range. This allows miners time to sell off/repurpose their hardware rather than a large amount of hashing power being put up for auction at the same time while people try to figure out what to do with their now "worthless" hardware. Additionally the decay promotes a known schedule of a deflationary curve of the next few years in prep for Proof of Stake consensus replacement of Proof of Work.
+## Specification
+Adjust Block, Uncle, and Nephew rewards
+Adjust the block reward to new_block_reward, where
+
+new_block_reward = 3_000_000_000_000_000_000 if block.number >= FORK_BLKNUM else block.reward
+(5E18 wei, or 3,000,000,000,000,000,000 wei, or 3 ETH).
+
+Analogue, if an uncle is included in a block for block.number >= FORK_BLKNUM such that block.number - uncle.number = k, the uncle reward is
+
+new_uncle_reward = (8 - k) * new_block_reward / 8
+This is the existing formula for uncle rewards, simply adjusted with new_block_reward.
+
+The nephew reward for block.number >= FORK_BLKNUM is
+
+new_nephew_reward = new_block_reward / 32
+This is the existing formula for nephew rewards, simply adjusted with new_block_reward.
+
```

Comment:
```suggestion
```
Looks like something got screwed up when you migrated from your previous PR.  This all shouldn't be here (actual spec is below).